runbotics/runbotics-desktop/src/action/microsoft/excel/excel.service.ts

```diff
@@ -139,14 +140,29 @@ export class ExcelService {
 
     /**
      * @see https://learn.microsoft.com/en-us/graph/api/range-update?view=graph-rest-1.0&tabs=javascript
+     * @description {string} url requires range address in a format of column letters, e.g. "A1:C3"
      */
-    setRange(session: ExcelSession, address: string, values: string | ExcelCellValue[][]) {
-        const url = `/worksheets/${session.worksheetName}/range(address='${address}')`;
-
+    setCells(session: ExcelSession, startCell: string, values: ExcelCellValue[][]) {
         if (!Array.isArray(values)) {
             values = JSON.parse(values) as ExcelCellValue[][];
```

Comment:
if input won't be array the JSON will throw an error. consider wrapping this in try/catch and throwing   Error(CloudExcelErrorMessage.setCellsIncorrectInput());